protocols/imap/src/main/java/org/apache/james/imap/processor/AbstractSelectionProcessor.java

```diff
@@ -181,134 +182,8 @@ private void respond(String tag, ImapCommand command, ImapSession session, Mailb
                         uidSet = new UidRange[] {new UidRange(MessageUid.MIN_VALUE, uidNext.previous())};
                     }
                 }
-                
```

Comment:
could you do a commit for each extract, here it's impossible to follow because you did too many changes at once.